cdk/kittyhawk/src/application.ts

```diff
@@ -40,24 +40,26 @@ export class Application extends Construct {
  * @param envKey name of the environment variable to insert
  * @param envValue value of the environment variable
  */
-function insertIfNotPresent(envArray: { name: string; value: string }[], envKey: string, envValue: any) {
-  const envSettingsModule = envArray?.filter(env => (env.name === envKey));
-  if (envSettingsModule?.length > 0) {
+export function insertIfNotPresent(envArray: { name: string; value: string }[], envKey: string, envValue: any) {
+  const envSettingsModule = envArray.filter(env => (env.name === envKey));
+  if (envSettingsModule.length > 0) {
     throw new Error(`${envKey} should not be redefined as an enviroment variable.`);
   }
   envArray.push({ name: envKey, value: envValue });
 
 }
 export interface DjangoApplicationProps extends ApplicationProps {
   /**
-   * Domain of the application.
+   * List of domain(s) of the application and if they are a subdomain.
```

Comment:
I think we should probably make this a bit more clear as to when you would set `isSubdomain` to be true/false, but I honestly don't know a good way of phrasing it, since for almost all of our products it's just a measure of if you're a subdomain, but for h@p it shouldn't be marked as such

Reply:
clarified this a bit!